network/Message.h

```diff
@@ -51,13 +51,16 @@ namespace Moderator {
   * misbehave as well.) */
 class FO_COMMON_API Message {
 public:
-    enum Parts : std::size_t {TYPE = 0, SIZE, Parts_Count};
+    enum Parts : std::size_t {TYPE = 0, SIZE, CPR_USED, Parts_Count};;
```

Comment:
```suggestion
    enum Parts : std::size_t {TYPE = 0, SIZE, Parts_Count, CPR_USED};
```
new thing at end to maintain consistency of old values underlying representation

Reply:
I do not know if there is a way to add additional fields to the header without putting its identifier before `Parts_Count`; `Parts_Count` effectively sets the size of the header.

Reply:
@geoffthemedio It won't work that way.